pkg/flow/adapter/xmltojsontransformation/adapter.go

```diff
@@ -46,6 +47,10 @@ type envAccessor struct {
 	BridgeIdentifier string `envconfig:"EVENTS_BRIDGE_IDENTIFIER"`
 	// CloudEvents responses parametrization
 	CloudEventPayloadPolicy string `envconfig:"EVENTS_PAYLOAD_POLICY" default:"error"`
+	// Sink defines the target sink for the events. If no Sink is defined the
+	// events are replied back to the sender.
+	// +optional
```

Comment:
nit: This is a Kubernetes convention, but this struct is not related to a Kubernetes API.
In envConfig, things are always optional unless the `required` tag is set.